src/status_im/contexts/wallet/events/collectibles.cljs

```diff
@@ -122,7 +122,8 @@
          {:keys [collectibles]} response
          collectible            (first collectibles)]
      (if collectible
-       {:fx [[:dispatch [:wallet/store-last-collectible-details collectible]]]}
+       {:fx [[:dispatch [:wallet/store-last-collectible-details collectible]]
+             [:dispatch [:navigate-to :wallet-collectible]]]}
        (log/error "failed to get collectible details"
```

Comment:
Hey @mmilad75 

thanks for applying the suggestion, but I think we could delay the nvagation even more, inside the event that is storing:

![image](https://github.com/status-im/status-mobile/assets/90291778/a5e3e1c4-8a24-4402-9c6f-ff1b20999972)

wdyt?

Reply:
Sure 👍 Let me apply it

Reply:
Done @ulisesmac